packages/cli/internal/pkg/osutils/utils.go

```diff
@@ -89,3 +95,15 @@ func CopyFileRecursivelyToLocation(destinationDir string, sourceDir string) erro
 
 	return err
 }
+
+func getAndCreateRelativePath(currentPath string, sourcePath string, destinationDir string) (string, error) {
+	newFilePath := strings.ReplaceAll(currentPath, sourcePath, "")
```

Comment:
Wouldn't this break on same-named folders?

e.g. 

```go
currentPath := "/home/tneely/workflows/agc/project-root/workflows"
sourcePath := "workflows"

newFilePath := "/home/tneely//agc/project-root/"
```

Reply:
That isn't an accurate representation as all of the paths are absolute although I could make this clearer if you prefer. An example would be 

```
currentPath:    "/home/tneely/workflows/agc/project-root/workflows/file.ext",
sourcePath:     "/home/tneely/workflows/agc/project-root",
newFilePath: "workflows/file.ext",
```